### PR TITLE
Migration: Show plan price, themes, and plugins in the flow on upgrade step

### PR DESCRIPTION
**Pull request by jonathansadowski**
#### Changes proposed in this Pull Request

* Updates the Migration confirmation step
* Adds a list of themes and plugins

#### Testing instructions

* Apply the PR and run Calypso locally, or use `calypso.live`.
* Do a migration into a site without a Business plan.
* After clicking "Import everything" on the confirm page, you should see the new style of upgrade step page:

image

* On purchasing a plan and following the other steps the migration should complete.

### CODE REVIEW

File: client/my-sites/migrate/section-migrate.jsx [hunk 1]
```diff
@@ -60,15 +62,43 @@ class SectionMigrate extends Component {
...
+				this.setState( { sourceSitePlugins: data.plugins } );
+			}
+		} );
+
+		wpcom.undocumented().themes( this.props.sourceSite.ID, { apiVersion: '1' }, ( err, data ) => {
+			if ( data.themes ) {
+				const sourceSiteThemes = [
+					// Put active theme first
+					...data.themes.filter( theme => theme.active ),
+					...data.themes.filter( theme => ! theme.active ),
```
Comment (andfinally) [inline client/my-sites/migrate/section-migrate.jsx]: ❤️ Elegant!

**Comment by andfinally** [comment]
Just done two migrations – one on an AT site without a Business plan, another on a completely fresh site – and they worked fine, so ¯\_(ツ)_/¯. On a second go it seemed more of a good idea to make the plugin and theme name text smaller – certainly the "7 more" could be smaller.

image

**Comment by jonathansadowski** [comment]
Thanks for taking a look... yeah, I basically haven't styled those lines yet. I'm looking into the pricing issue at the moment. Notice in your screenshot that it shows you $20 per month, billed annually? That's not actually the price. For some reason it appears to be picking up on the fact that it's 20 something (GBP, I presume?) without showing the correct currency symbol. For me, it's neither showing the correct currency symbol, nor the correct price in that currency.

**Comment by andfinally** [comment]
Yes, I think it must be the currency, because when I actually pay the cart shows £240.

**Comment by jonathansadowski** [comment]
Thanks for the reviews!

**Pull request merged**